### ISSUE TITLE
59fps on Windows

### ISSUE THREAD

**Issue by CommodoreAmiga** [yquake2/yquake2#371]
Hi, I'm having issue that framerate was smoother on previous build with Windows 10.

### **Build 7.30:**
Windows 10: 60fps
MacOS Mojave: 60fps

### **Build 7.40:**
Windows 10: 59fps
MacOS Mojave: 60fps

I have Windows and Mac on same machine, GPU is GeForce 750m, running with same settings.

**Comment by Yamagi** [comment]
That's not a bug but a feature. ;) The bigger problem here is that Windows can't handle fractional display refresh rates. You've likely got a 59.95hz display, so the driver is forced to round the value up or down. Most drivers round up, but some - like yours round - down...  
If the driver rounds up everything is okay. 0.05hz too much is something that Quake II can cope with. If the driver round down the communicated framerate is 0.95hz too low, that's too much offset for Quake II. Up to 7.30 we increased all(!) display refresh rates by 1, so your 59hz became 60hz and everything was good. But if the driver rounded up 60hz became 61hz and thing broke. Therefor we've decided to stick with whatever Windows communicates to us and introduce a new cvar `vid_displayrefreshrate` that can override the communicated display refresh rate.

TL;DR: Set `vid_displayrefreshrate` to 60 and you're done. :)

**Comment by CommodoreAmiga** [comment]
Thanks for help, I set vid_displayrefreshrate to 61 and now it's smooth (cl_showfps says 60fps) on Windows.
